docs/ext/commands/help.rst

```diff
@@ -0,0 +1,78 @@
+.. currentmodule:: discord
+
+.. _ext_commands_help:
+
+Help Commands
+=================
+
+The simplicity of :class:`~.commands.HelpCommand` allows you to customise
+your experience really easily.
+
+Sending Help
+---------------
+
+You may want to send help if the user doesn't use your command
+right. Here is an example of what you can do:
+
+.. code-block:: python3
+    :caption: sos.py
+    :emphasize-lines: 8, 10
+
+    from discord.ext import commands 
+
+    @commands.command()
+    async def sos(ctx, what: str=None):
+        """Send Help!"""
+        if not what:
+            return await ctx.send_help('sos')
+        await ctx.send(f'Sending help for {what}!')
```

Comment:
Remember that Python 3.5 is still supported by this library and does not support f-strings.